doc/guides/prog_guide/cnet.rst

```diff
@@ -0,0 +1,389 @@
+..  SPDX-License-Identifier: BSD-3-Clause
+    Copyright (c) 2022 Intel Corporation.
+
+.. _CNET_Library:
+
+CNET Stack Library
+==================
+
+Overview
+--------
+
+The CNET graph node library uses a set of graph nodes to create a UDP/TCP/IPv4 network stack in
+user space. One goal is to provide a network stack for applications in CNDP as the developer
+may have been using the Linux network stack via the socket interface.
+
+The CNET stack uses a socket like interface called channels to make it fimiliar to the developer,
+but does have a number of differences. One of the differences is that channels support multiple
+packets interface using a callback method to receive and a multiple packet transmit API.
+
+.. note:: CNET Stack *TCP* support is **experimental** for release 22.08.
+
+.. _figure_cndp_system_overview:
+
+.. figure:: img/cndp_system_overview.*
+
+   CNDP System Overview
+
+A CNDP application is running in user-space in a container or bare-metal environments and provides
+a set of libraries for the application developer to quickly create a *Cloud Native* application for
+more details and information about CNDP please refer to the :ref:`CNDP Overview <CNDP_Overview>`.
+
+A number of libraries are available in CNDP and some are used in the CNET stack to provide better
+performance and functionality for the application developers please refer to
+:ref:`CNDP Components <CNDP_Components>` as needed.
+
+.. note:: The *cnet-graph* example is a reasonable source on how to create an application that
+  uses the CNET graph nodes. The *l3fwd-graph* example uses a different set of graph nodes then
+  CNET stack, but provides another solution for a very simple packet processing application.
+
+CNET POD/Container Overview
```

Comment:
`Pod` should not be in all capitals. It is not an acronym, it simply means `pod` as in `pea pod`